templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -524,8 +526,8 @@ contents:
         rollback_nm
         print_state
       fi
-      
-      convert_to_bridge "$iface" "br-ex" "phys0"
+
+      convert_to_bridge "$iface" "br-ex" "phys0" "${BRIDGE_METRIC}"
```

Comment:
conflict: remove mentions of br-ex 1 and phys1 as 4.8 has no support for secondary bridge in configure-ovs.